styles.css

```diff
@@ -1,60 +1,162 @@
-*{
-  margin: 0;
-  padding: 0;
-
-}
-
-.top-nav {
-  background-color: #1C1A19;
-  display: flex;
-  justify-content: space-between;
-    
-
-}  
-.left {
-  color: #FFFFFF;
-  padding: 15px 150px;
-  font-family: 'Inter';
-  font-style: normal;
-
-} 
-.right {
-    padding: 15px 150px;
-    line-height: 20px;
-}
-.right a{
-  color: #FFFFFF;
-  padding: 10px;
-  text-decoration: none;
-  
-  
-    
-} 
-.right i{
-  color: #FFFFFF;
-  text-decoration: none;
-  
-}
-.main {
-  display: grid;
-  grid-template-columns: repeat(3, 1fr);
-  grid-auto-rows: 1fr;
-  border: 1px solid green;
-  padding: 250px 50px;
-  color: #FFFFFF;
-  background-color: black;
-  /* display: flex;
-  align-items: center;
-  justify-content: space-between;
-  
-  
- 
-  
-  
-}
-.left-side ul li {
-  list-style-type: none;
-  
-  
-  
-}
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #191765;
+  width: 100vw;
+  height: 100vh;
+}
+
+.header {
+  width: 100%;
+  height: max-content;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 30px;
+}
+
+.nav-links {
+  display: none;
+}
+
+.logo {
+  text-decoration: none;
+  color: #fff;
+  text-transform: capitalize;
+  font-size: 20px;
+}
+
+.fa-bars {
+  color: #fff;
+  font-size: 20px;
+}
+
+.main {
+  width: 90%;
+  margin: 50px auto 950px auto;
+  height: max-content;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.image {
+  width: 110px;
+  position: absolute;
+  top: 50px;
+  left: 0;
+}
+
+.image img {
+  width: 100%;
+}
+
+.image-2 {
+  width: 50px;
+  position: absolute;
+  top: 500px;
+  right: 0;
+}
+
+.image-2 img {
+  width: 100%;
+}
+
+.main-intro {
+  width: 80%;

```

Comment:
Kindly remove this width that way your design will match the Figma design.


| Your Design | Figma Design |
|-------------|---------------|
| <img width="375" alt="image" src="https://user-images.githubusercontent.com/34578631/171230854-c999d525-3c86-4ec4-8311-80b4f44a0320.png"> | <img width="378" alt="image" src="https://user-images.githubusercontent.com/34578631/171230911-6026cda5-424c-47e3-a4c2-3f6833ae30ef.png"> |

Reply:
- [ ] As mentioned by the previous reviewer, please remove the width property for the `.main-intro` class to match the design.